debian/patches/runpm-workaround.patch

```diff
@@ -0,0 +1,43 @@
+--- a/share/hybrid/gpu-manager.c
++++ b/share/hybrid/gpu-manager.c
+@@ -1418,6 +1418,32 @@
+     unlink(xorg_d_custom);
+ }
+ 
++/*
++ * Work around broken power manangement in Linux and NVIDIA drivers.
++ * See: https://download.nvidia.com/XFree86/Linux-x86_64/440.31/README/dynamicpowermanagement.html#KnownIssuesAndW6426e
++ */
++static void remove_extra_pci_functions(const struct device *device) {
++    fprintf(log_handle, "Applying workaround for power management\n");
++    for (unsigned int fn_nr = 1; fn_nr <= 3; ++fn_nr) {
```

Comment:
Shouldn't all functions be removed? That means all the way up to fn_nr 7

Reply:
The docs says NVIDIA GPUs only have up to 4 functions.

Reply:
So far, yes. But that could always increase. The maximum for one PCIe device is 8 functions